model/src/main/resources/reference.yml

```diff
@@ -40,6 +40,7 @@ ort:
   # configurations for the same package, see https://github.com/oss-review-toolkit/ort/issues/6972 for details.
   packageConfigurationProviders:
   - type: DefaultDir
+  - type: OrtConfig
```

Comment:
Nit: Maybe make `OrtConfig` last in the list to keep `DefaultDir` and `Dir` next to each other?

Reply:
Yes, I also think that this makes more sense. Fixed.